google-cloud-storage/src/main/java/com/google/cloud/storage/XmlObjectParser.java

```diff
@@ -67,13 +68,18 @@ public Version version() {
           public void setupModule(SetupContext context) {
             context.addSerializers(
                 new SimpleSerializers(
-                    ImmutableList.of(new StringEnumValueSerializer<>(StorageClass.class))));
+                    ImmutableList.of(
+                        new StringEnumValueSerializer<>(StorageClass.class),
+                        new StringEnumValueSerializer<>(ObjectLockMode.class))));
```

Comment:
Is something similar required for PredefinedAcl?

Reply:
PredefinedAcl is a java enum, not a StringEnumValue, for that we would need to add both a new serializer and deserializer, which would be more appropriate for a separate PR.